collection-scripts/gather_monitoring

```diff
@@ -101,14 +105,15 @@ monitoring_gather(){
   # begin gathering
   # NOTE || true ignores failures
 
-  prom_get rules rules   || true
-  prom_get alertmanagers alertmanagers  || true
-  prom_get status/config status/config || true
-  prom_get status/flags status/flags || true
-  prom_get status/runtimeinfo status/runtimeinfo || true
-  prom_get_from_replicas status/tsdb status/tsdb || true
+  prom_get rules    || true
+  prom_get alertmanagers   || true
+  prom_get status/config  || true
+  prom_get status/flags  || true
+  prom_get targets  || true
```

Comment:
I'd recommend that the script gathers from both replicas since they may not have the same view. I would also limit the query to active targets.

```suggestion
  prom_get_from_replicas targets?state=active active_targets  || true
```

Reply:
@simonpasquier thank you for the review. 

Should `prom_get_from_replicas` be used for all resources that we are collecting? 
There is also `alertmanager_get` does that need to change as well to collect from individual pods?